src/main/java/org/splitec/service/CosmosService.java

```diff
@@ -0,0 +1,49 @@
+package org.splitec.service;
+
+import com.azure.cosmos.CosmosContainer;
+import com.azure.cosmos.models.CosmosQueryRequestOptions;
+import com.azure.cosmos.util.CosmosPagedIterable;
+import org.splitec.client.DatabaseClient;
+import org.splitec.model.User;
+
+import java.util.concurrent.atomic.AtomicReference;
+
```

Comment:
@luucasfl Please add the annotation to Service bean

Reply:
Im not a java super programmer to do this